app/helpers/users_helper.rb

```diff
@@ -1,2 +1,7 @@
 module UsersHelper
+  def gravatar_for(user)
+    gravatar_id = Digest::MD5::hexdigest(user.email.downcase)
```

Comment:
__[reek]__
* [FeatureEnvy] refers to user more than self (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Feature-Envy.md)